.github/workflows/flake-updates.yml

```diff
@@ -15,6 +15,11 @@ jobs:
           extra_nix_config: |
             experimental-features = nix-command flakes
       - name: Update flake.lock
+        id: update
         uses: DeterminateSystems/update-flake-lock@v19
         with:
           token: ${{ secrets.GH_TOKEN_FOR_UPDATES }}
+      - name: Enable Automerge
+        run: gh pr merge --rebase --auto "${{ steps.update.outputs.pull-request-number }}"
```

Comment:
Probably not a concern for this repository but how does merge queues work with pull requests from non-committers? Bors performs a snapshot of the code before it merges while merge queue seems to allow changes after being enabled.

Reply:
> merge queue seems to allow changes after being enabled.

I haven't actually used the merge queue yet so I'll have to check this, I had assumed that it wouldn't allow it.

Reply:
I think if the PR is modified it's removed from the queue?